playbook/app/pb_kits/playbook/pb_radio/_radio.jsx

```diff
@@ -40,7 +42,7 @@ const Radio = ({
 }: RadioProps, ref) => {
   const ariaProps = buildAriaProps(aria)
   const dataProps = buildDataProps(data)
-  const classes = classnames(buildCss('pb_radio_kit'), { error }, { dark }, globalProps(props), className)
+  const classes = classnames(buildCss('pb_radio_kit'), { error }, { dark }, globalProps(props), className, alignment)
```

Comment:
We need className last in this list because of CSS Modules stuff in nitro.

Reply:
Thanks, good catch!